### PR TITLE
Add boilerplate media type definitions for each protocol message.

### LINKED ISSUE #105: DefineMIME types for all of the things we are carrying over HTTP

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #105

### CODE REVIEW

**Comment by chris-wood** [comment]
@mnot -- is it better practice to define media types per protocol message, or to define a single media type for all protocol messages?

**Comment by mnot** [comment]
Generally, if they're different formats -- e.g., they have different schema, or just different syntax -- they need different media types (or an explicitly generic one like `application/octet-stream`). If it's one format that has different uses, you can give it a single media type and then switch contexts based upon internal flags.

The real question, though, is whether you have any potential for use cases where it's important for generic software that's handling these messages to do different things based upon the media type.

**Comment by chris-wood** [comment]
> Generally, if they're different formats -- e.g., they have different schema, or just different syntax -- they need different media types (or an explicitly generic one like application/octet-stream). If it's one format that has different uses, you can give it a single media type and then switch contexts based upon internal flags.
>
> The real question, though, is whether you have any potential for use cases where it's important for generic software that's handling these messages to do different things based upon the media type.

I don't think there'd ever be generic software using these types (beyond the HpkeConfig?). Most messages are meant to be consumed by PPM and PPM alone. I moved protocol messages to the "message" registry space and kept the HpkeConfig in the application type space, and also left all intended usage fields as COMMON. I also added an open issue to have someone more familiar with these registries give these allocation requests a review.

@ekr, this should be good to go based on the above.

File: draft-pda-protocol.md [hunk 1]
```diff
@@ -495,7 +495,7 @@ keys for a reasonable period after key changes in order to avoid
 rejecting reports.
 [[OPEN ISSUE: https://github.com/abetterinternet/prio-documents/issues/106]]
 
-### Upload Request
+### Report
```
Comment (ekr) [inline draft-pda-protocol.md:498]: This hed seems a bit short. What's your thinking here.

**Review by ekr — APPROVED** [review]
LGTM except for the hed.

File: draft-pda-protocol.md (hunk 1 above)
Comment (chris-wood) [reply draft-pda-protocol.md:498]: Only that it was more clear, but it's a separate change, so I'll just revert.

**Pull request merged**